freebsd/FreeBSDProcessList.c

```diff
@@ -444,6 +444,7 @@ static void FreeBSDProcessList_updateProcessName(kvm_t* kd, const struct kinfo_p
    *at = '\0';
 
    Process_updateCmdline(proc, cmdline, 0, end);
+   free(cmdline);
```

Comment:
I'd suggest to have a blank line to visually structure things a bit more.
```suggestion

   free(cmdline);
```
Please squash that change into your existing commit.

Apart from this LGTM.

Reply:
Is this OK? Did I do it right?

Reply:
The second change itself is correct, but it would be nicer to have both lines changed as part of the same commit. Have a look at `git rebase -i` and the `fixup` operation. The result should be just one commit with both lines changed.

Reply:
I hope I now managed to squash what was once HEAD~2.